docs/source/markdown/podman.1.md

```diff
@@ -73,18 +76,21 @@ When namespace is set, created containers and pods will join the given namespace
 **--network-cmd-path**=*path*
 Path to the command binary to use for setting up a network.  It is currently only used for setting up a slirp4netns network.  If "" is used then the binary is looked up using the $PATH environment variable.
 
-**--remote**, **-r**=*url*
-URL to access Podman service (default "unix:/run/user/3267/podman/podman.sock")
+**--remote**, **-r**
+Access Podman service will be remote
+
+**--url**=*value*
+URL to access Podman service (default from `containers.conf` or lastly "unix:/run/user/3267/podman/podman.sock").
```

Comment:
Can we be more verbose about the default? Root doesn't look there, and we should say that we use `/run/user/$UID` to be completely clear.